### ISSUE TITLE
Can't type in encrypted group chat

### ISSUE THREAD

**Issue by colegleason** [WhisperSystems/TextSecure#3036]
Phone details: HTC One m8 running CM 12 nightly 4/10/2015. TextSecure 2.11.1

Debug Log: https://gist.github.com/anonymous/f7429f23e8760d6ffdb6

I was added to an encrypted group chat with other TextSecure users, but I cannot open the keyboard. I am using the standard CM12 keyboard. If I press the emoji key, I can add emojis, but I can't send the message. From the emoji keyboard, I can switch and open the normal Latin keyboard, but if I press any letter, the characters do not appear in the text area. 

I can receive messages in the group chat. I can also type fine in one-on-one SMS or TextSecure messages.

I tried to pull a video off the phone, but TextSecure just shows up as a black screen under `adb screenrecord`.

**Comment by rhodey** [comment]
@colegleason can you ask one of your friends in this group to make an update to the group either by adding a member, changing the title, or changing the group icon? I believe that should correct the symptoms of this bug.

**Comment by colegleason** [comment]
That fixed it! Thank you.

**Comment by rhodey** [comment]
@colegleason to clarify, can you confirm that you were unable to enter text **from the moment you were added to the group**, and not after some strange uninstall/re-install or backup sequence? thanks :)

**Comment by colegleason** [comment]
I think this was after I reinstalled TextSecure after changing from stock
HTC OS to CM12, so it was not a new group. I could type before I did the
install of CyanogenMod.

So this was after a strange install sequence.

**Comment by rhodey** [comment]
thanks @colegleason, closing and linking to #2581 for reference.

**Comment by Ralms** [comment]
How is this still not fixed in 2018? 

I just switched phones and this exact issue happen with the Signal App in Android.

**Comment by haffenloher** [comment]
@Ralms it should be. If it doesn't work for you, could you please open a new issue and include exact reproduction steps and a debug log? Thanks!

**Comment by Ralms** [comment]
@haffenloher When I've setup my new phone, all groups were gone because I didn't do backup from the old phone, but the desktop client retained everything. After I switch phones the desktop client asked to relink which I did and kept all conversations and groups. 

At this point I wad able to type messages in Desktop, but when I went to my phone I was unable to, only after asking a friend to rename the group and there ir started working. Essentially I has to use the work around.

There are a lot of minor issues with the apps such as this, that make it hard to convince people to switch from WhatsApp unfortunately.
